R/utils.R

```diff
@@ -108,3 +108,28 @@ rasterize_if_needed <- function(plot) {
   plot <- if(getOption("dualsimplex-rasterize", default=FALSE)) ggrastr::rasterise(plot, dpi=600) else plot
   return(plot)
 }
+
+# TODO (cjlee): Modify this function for only using truncated SVD
+calc_dist_from_truncated_svd <- function(scaled, projected) {
```

Comment:
Can't we just calculate the distance between scaled matrix and "reconstructed scaled matrix based on first K components"